diff -Nruw ncpfs/dkms.conf ncpfs.SKIP_STACK_VALIDATION/dkms.conf
--- ncpfs/dkms.conf	2019-04-24 09:44:33.000000000 +0200
+++ ncpfs.SKIP_STACK_VALIDATION/dkms.conf	2019-04-24 11:18:14.064026451 +0200
@@ -1,6 +1,6 @@
 PACKAGE_NAME="ncpfs"
 PACKAGE_VERSION="2.2.6"
-MAKE="KDIR=/lib/modules/$kernelver/build MDIR=/lib/modules/$kernelver make"
+MAKE="KDIR=/lib/modules/$kernelver/build MDIR=/lib/modules/$kernelver make SKIP_STACK_VALIDATION=1"
 CLEAN="make clean"
 BUILT_MODULE_NAME[0]="ncpfs"
 AUTOINSTALL="yes"
